application/server/main/tasks.py

```diff
@@ -112,81 +113,99 @@ def create_task_unpaywall(args):
         logger_console.debug(db_handler.fetch_all())
 
 
-def get_softcite_version(local_files):
-    softcite_files = [file for file in local_files if file.endswith('.software.json')]
-    if softcite_files:
-        with open(softcite_files[0], 'r') as f:
-            softcite_version = json.load(f)['version']
-    else:
-        softcite_version = "0"
+def get_softcite_version(softcite_file: str) -> str:
```

Comment:
Could this arg be renamed "softcite_file_path" just to be clear that it is not a binary file but a path ?